workers/tasks/export/html.tsx

```diff
@@ -287,9 +288,13 @@ export const renderStaticHtml = async (options: RenderStaticHtmlOptions) => {
 		renderedStructuredValues,
 	});
 
-	const renderableNodes = [filterNonExportableNodes, addHrefsToNotes, blankIframes]
-		.filter((x): x is (nodes: any) => any => !!x)
-		.reduce((nodes, fn) => fn(nodes), pubDoc.content);
+	const renderableNodes = pagedTarget
+		? [filterNonExportableNodes, addHrefsToNotes, blankIframes]
+				.filter((x): x is (nodes: any) => any => !!x)
+				.reduce((nodes, fn) => fn(nodes), pubDoc.content)
+		: [filterNonExportableNodes, addHrefsToNotes]
+				.filter((x): x is (nodes: any) => any => !!x)
+				.reduce((nodes, fn) => fn(nodes), pubDoc.content);
```

Comment:
I tried to DRY this but these node traversing functions are called recursively in the reduce function, so you can't just run one at a time.